grafana/dashboards/bookkeeper.json

```diff
@@ -383,7 +383,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "bookkeeper_server_ADD_ENTRY_REQUEST{cluster=~\"$cluster\", kubernetes_pod_name=~\"$bookie\", success=\"true\", quantile=\"0.5\"}",
+              "expr": "bookkeeper_server_ADD_ENTRY{cluster=~\"$cluster\", kubernetes_pod_name=~\"$bookie\", success=\"true\", quantile=\"0.5\"}",
```

Comment:
check for the existence of ADD_ENTRY_REQUEST and its two metrics, so don't need change this, right?

Reply:
![image](https://github.com/apache/pulsar/assets/15684156/7d6b1bfd-2829-4a35-bf72-5b2fbf359b46)
Both metrics are exist, but the value is `NaN`.

Marked as abandoned in the [release note](https://github.com/apache/bookkeeper/releases/tag/release-4.16.0):
![image](https://github.com/apache/pulsar/assets/15684156/4e781538-57ca-4db7-a729-3006f51e227b)

Reply:
if use V1, it can work? it seem not abandon, only don't support in V2 protocol right?

Reply:
Yes, it appears that the upgrade to 3.0 has already switched to the v2 protocol, so now the user is based on the master branch `bookkeeper.json` is unable to view the corresponding indicator; 

Users with older versions of `bookkeeper.json` using the corresponding branch can work properly.